package.json

```diff
@@ -41,6 +41,7 @@
     "figures": "^3.1.0",
     "fs-extra": "^8.1.0",
     "ghauth": "^4.0.0",
+    "https-proxy-agent": "^5.0.0",
```

Comment:
Why not use https://www.npmjs.com/package/proxy-agent to support more protocols?